src/includes_normalize-win32.cc

```diff
@@ -96,8 +96,12 @@ string IncludesNormalize::Relativize(StringPiece path, const string& start) {
 
 string IncludesNormalize::Normalize(const string& input,
                                     const char* relative_to) {
-  char copy[_MAX_PATH];
+  char copy[_MAX_PATH + 1];
```

Comment:
There's other use of MAX_PATH in this file.  Maybe it'd be good to wrap GetFullPathName as a helper function so that none of the other code is manually managing buffers?